beets/ui/__init__.py

```diff
@@ -471,29 +477,23 @@ def get_replacements():
     return replacements
 
 
-def _pick_format(album, fmt=None):
-    """Pick a format string for printing Album or Item objects,
-    falling back to config options and defaults.
-    """
+def format_(obj, fmt):
+    """Print a object, intended for Album and Item
+
+    This is equivalent to `format`, but the spec can be None
+    `fmt` is mandatory for otherwise one can just call `format(my_object)`"""
```

Comment:
You're definitely right, in my mind had formed the bizarre conception that `format(item, '')` would only produce empty strings. I've removed `format_` -- and with that `print_obj`. I mostly had to change default value of `--format` options and that was enough.